src/erlcloud_iam.erl

```diff
@@ -328,6 +329,26 @@ list_attached_role_policies(RoleName, PathPrefix, #aws_config{} = Config)
 %
 % Policies API
 %
+-spec(list_policies/0 :: () -> proplist()).
+list_policies() ->
+    list_policies("/").
+-spec(list_policies/1 :: (string() | aws_config()) -> proplist()).
+list_policies(#aws_config{} = Config) ->
+    list_policies("/", Config);
+list_policies(PathPrefix) ->
+    list_policies(PathPrefix, default_config()).
+
+-spec(list_policies/2 :: (string(), aws_config()) -> proplist()).
+list_policies(PathPrefix, #aws_config{} = Config) ->
+    list_policies(PathPrefix, false, default_config()).
+
+-spec(list_policies/3 :: (string(), boolean(), aws_config()) -> proplist()).
+list_policies(PathPrefix, OnlyAttached, #aws_config{} = Config)
```

Comment:
Probably it is better to pass request parameters in a list rather than putting them into function definition.